docs/dictionary/command/create.lcdoc

```diff
@@ -4,7 +4,7 @@ Synonyms: new
 
 Type: command
 
-Syntax: create [invisible] <objectType> [<objectName>] [in <group>]
+Syntax: create [invisible] <objectType> [<objectName>] [in <group>] [in <card>]
```

Comment:
Sorry, I gave you bad advice here - it needs to be 
`[in {<group> | <card>}]` as things that are parameters to the syntax have to be enclosed in `<>`